lib/model/course/course.dart

```diff
@@ -136,111 +212,6 @@ class CourseAvailabilityUtility {
   }
 }
 
-//class FacultyUtility {
```

Comment:
Hmm I wanted to keep this block to change from Faculty string to Facutly enum. But OK lets keep this change.